policy/modules/services/dovecot.if

```diff
@@ -61,6 +61,85 @@ interface(`dovecot_domtrans_deliver',`
 	domtrans_pattern($1, dovecot_deliver_exec_t, dovecot_deliver_t)
 ')
 
+########################################
+## <summary>
+##	Read and write dovecot unix
+##	stream sockets.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dovecot_rw_stream_sockets',`
+	gen_require(`
+		type dovecot_t;
+	')
+
+	allow $1 dovecot_t:unix_stream_socket { connectto rw_stream_socket_perms };
```

Comment:
I suspect that the rw_stream_socket_perms is a bit too much. Would just `rw_socket_perms` work instead?

Reply:
> I suspect that the rw_stream_socket_perms is a bit too much. Would just `rw_socket_perms` work instead?

Let me try it. These are rules coming from one of my machines that is set up in this way.

Reply:
After some more testing, the only permissions needed by `postfix_pipe_t` here seems to be `connectto` and `write`. So it looks like we can reduce this access to `rw_socket_perms` as you said.